core/src/main/resources/error/error-classes.json

```diff
@@ -1729,7 +1718,8 @@
       },
       "WITH_SUGGESTION" : {
         "message" : [
-          "Consider to change the number of arguments because the function requires <expectedNum> parameters but the actual number is <actualNum>."
+          "The <functionName> requires <expectedNum> parameters but the actual number is <actualNum>.",
+          "If you have to call this function with <legacyNum> parameters, set the legacy configuration <legacyConfKey> to <legacyConfValue>."
```

Comment:
You are repeating the general error class message.

Reply:
Thanks for the correction! Just fixed it.